installer/README.md

```diff
@@ -21,31 +21,48 @@ Testing on other major cloud providers is in the roadmap.
 To use the Cloudflow GKE installer, you need to have the following packages installed on your local machine:
 
 * [Google Cloud SDK](https://cloud.google.com/sdk/)
+* [Amazon CLI for EKS](https://eksctl.io/)
```

Comment:
I suggest we rephrase this to state something like "the configuration command line tool for your particular Kubernetes cluster, like:" and then have a sub-list of the supported tools.

Reply:
Fixed